### PR TITLE
Printing privileges added

### PR DESCRIPTION
**Pull request by OsnaFonz**
as mentioned in issue #107

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 * {
-	font-size: 12px;
+	font-size: 14px;
```
Comment (Arachnid) [inline css/main.css:2]: Why are you adjusting the size and spacing for the entire page?

File: css/main.css [hunk 2]
```diff
@@ -407,6 +401,9 @@ table th {
 
 	#worldno span { padding:0; }
 	
-	.cards { display:none; }
+	#target_privileges { display:none; }
```
Comment (Arachnid) [inline css/main.css]: Please try and stick to camelCase or hypen-separated for class names.

File: css/main.css (hunk 1 above)
Comment (OsnaFonz) [inline css/main.css:2]: first of all 14px is the standard size for bootstrap and all styling for the theme are orientated to this size. and the menu had some styling issues with the altered font-size. 
and i think it looks a little bit nicer. especially on smart phones and tablets.

Comment (Arachnid) [inline css/main.css:2]: Okay, but changes like this - particularly anything that affects the styling of the entire document - should be isolated in a separate PR, so they can be reviewed independently.

**Comment by SteveOliverC** [comment]
Did this update get committed? I don't see privileges when I print out the rules for World 123.
